rfc-028-adopt-a-code-of-conduct.md

```diff
@@ -0,0 +1,138 @@
+
+- the standards of behaviour which are expected of everyone who interacts with
+  the project
+- who’s responsible for enforcing the standards
+- how they’ll enforce the standards – the consequences of violating the
+  standards
+
+We want to limit the scope of who finds out about a code of conduct report, to
+maintain privacy. So we’ll limit it to the Chief Technology Officer and
+Managing Director.
+
+### Guidelines
+
+All public repositories in the `dxw` organisation MUST contain a `CONTRIBUTING.md` and `CODE_OF_CONDUCT.md` file.
+
+The `CONTRIBUTING.md` file:
+
+- SHOULD explain whether the project is actively seeking contributions from
+  outside dxw
+- MUST link to the `CODE_OF_CONDUCT.md` file
+
+The `CODE_OF_CONDUCT.md` file:
+
+- SHOULD be a copy of the [Markdown version of the Contributor Covenant](https://www.contributor-covenant.org/version/2/0/code_of_conduct/).
+- SHOULD be modified to include the following information about contact methods:
+  > Instances of abusive, harassing, or otherwise unacceptable behavior may be
+  > reported to dxw’s Chief Technology Officer at code-of-conduct@dxw.com. If a
+  > complaint relates to the conduct of the Chief Technology Officer, then
+  > complaints may be reported to the Managing Director at
+  > code-of-conduct-md@dxw.com. The current holders of these roles can be found
+  > on [dxw’s people page](https://www.dxw.com/our-people/).
+
+When starting a client project, we SHOULD speak to the client to find out
+whether they would like us to add a code of conduct to the project. We’ll need
+to understand whether they:
+
+- have the ability to accept reports of violations
+- can do this privately
+- want to take on the responsibility of enforcing the code of conduct
+
+dxw SHOULD NOT take on the responsibility of enforcing the Code of Conduct for
+a client project.
+
+TODO Something about enforcement? – this is one to check with Dom and Dave too.
+This is from the Contributor Covenant site:
+
+> Do not simply add the Contributor Covenant to your project and assume that
+> any problems with civility, harassment, or discrimination will be solved.
+> As a project maintainer you must be committed to enforcing the code of
+> conduct. A code of conduct without enforcement sends a false signal that
+> your project is welcoming and inclusive, and can create a dangerous
+> situation for marginalized people who participate. Adding the Contributor
+> Covenant to a project places responsibility on the project team that must
+> not be taken lightly.
+>
+> Before adopting the Contributor Covenant take the time to discuss and
+> decide how to deal with problems as they emerge. Document the policy and
+> procedure for enforcement, and add it to your README or in another visible,
+> appropriate place. Consider if your project team has the willingness and
+> maturity to follow through on your enforcement procedures.
+
+## Concerns
+
+Adding a `CONTRIBUTING.md` might suggest that we’re actively seeking
```

Comment:
```suggestion
Adding a code of conduct might suggest that we’re actively seeking
```

Reply:
I think the contributing doc is the one which explains how people can contribute to the project, and so it’s the one which would imply we’re looking for contributions?

Reply:
What we're sugesting is addnig a `CONTRIBUTING.md` that explicitly states that we're not accepting contributions, though, aren't we?

Reply:
True – I've changed it.